sql/core/src/main/scala/org/apache/spark/sql/SQLConf.scala

```diff
@@ -100,6 +101,33 @@ private[spark] object SQLConf {
         }
       }, _.toString, doc, isPublic)
 
+    def intMemConf(
```

Comment:
I will make the changes. I used the code format and run the scalastyle test, thought it passed the styles. I will look more carefully next time. Thanks.